.codespellrc

```diff
@@ -0,0 +1,8 @@
+[codespell]
+skip = .git,*.pdf,*.svg,external,src
+ignore-words-list = ans,siz,homogenous,pth,nam,ext,dum,siz,toi,whos,als
+# case sensitive etc as regex
+ignore-regex = \b(ALS|correctT)\b|n\(ot a
```

Comment:
I cannot make sense of this regexp, it seems syntax-wise wrong.

Reply:
it is python regex, https://docs.python.org/3/library/re.html what in syntax confuses you?

Reply:
I looked it up. 

I was not familiar with `\b`, thought it was a backspace - now I know.

The `n\(ot a` confused me, I thought the `(` was a meta character at the start of a section, but now I see that it is to match `n(ot a landmark)`.

Thakns, all clear now!

PS I have solved it instead with a few more words to ignore, and with a `.codespellignorelines` exclude file.